### ISSUE TITLE
[RyuJIT/ARM32] Assertion failed '((consume == 0) && (produce == 0)) || (ComputeAvailableSrcCount(tree) == consume)'

### ISSUE THREAD

**Issue by sjsinju** [dotnet/coreclr#12733]
```
Assert failure(PID 592 [0x00000250], Thread: 592 [0x0250]): Assertion failed '((consume == 0) && (produce == 0)) || (ComputeAvailableSrcCount(tree) == consume)' in 'JitTest.LCS:Main():int' (IL size 279)
               
  File: /opt/code/github/hqueue/coreclr/src/jit/lsra.cpp Line: 3620
  Image: /home/dotnet_jenkins/workspace/hqueue/Run_coreclr_tests_Checked_ryujit_nyi_all/runtime/tests/Tests/coreoverlay/corerun
```

```
JIT/Methodical/refany/_il_dbglcs/_il_dbglcs.sh
JIT/Methodical/refany/lcs/lcs.sh
```

**Comment by sjsinju** [comment]
@BruceForstall Please assign this to me.

**Comment by sjsinju** [comment]
Fixed with #12628
